tools/hangar/src/paths.ts

```diff
@@ -5,6 +5,7 @@ export const repoRoot = path.resolve(__dirname, "../../..");
 export const examplesDir = path.join(repoRoot, "examples");
 export const testDir = path.join(examplesDir, "tests");
 export const validTestDir = path.join(testDir, "valid");
+export const appWithParamsDir = path.join(validTestDir, "apps_with_params");
```

Comment:
We have a couple other folders in this dir that do something similar so we can probably leave out the `app_with_` part of this and just call the folder `parameters`.
I don't have a strong opinion on this though because examples folder is really due for a big makeover anyways.